server/boot/challenge.js

```diff
@@ -36,7 +38,11 @@ function buildUserUpdate(
         completedChallenge: challengeId
       }
     };
-    finalChallenge = completedChallenge;
+    finalChallenge = {
+      ...completedChallenge,
+      lastUpdated: completedChallenge.completedDate,
```

Comment:
This line is unnecessary here. It is already part of `completeChallenge` and does not need to be re-added